### DISCUSSION TITLE
Suggestion: Using Mosquitto Repository instead Debian Repository

### DISCUSSION THREAD

**Discussion by skl283** [tteck/Proxmox#1736] [Ideas]
Hi @tteck,

thank you again for your great scripts! As already mentioned [here](https://github.com/tteck/Proxmox/issues/1735) i'll suggest to use the Mosquitto Repository for Debian.
I have a suggestion of using Mosquitto in latest Version: in yor scripts you using the Debian Repo, but mosquitto delivers its own repository server - also for bookworm.

please look at https://mosquitto.org/blog/2013/01/mosquitto-debian-repository/ - also a bookworm repo file is availale.

br
sash

**Comment by tteck** [comment]
During the Bookworm release, repositories for both MQTT and MariaDB were eliminated. https://github.com/tteck/Proxmox/commit/2101583d1a07f3a22eced4b41d95b2b0139e68dd https://github.com/tteck/Proxmox/commit/4d6a9b777d7d3321f85aad908f4d9a9698a86959

Considering that MQTT is one of those applications that typically "just works," I find the Debian-provided version to be appropriate. However, I'm still undecided about whether to reintroduce the repositories.

**Reply by tteck** [reply]
Reintroduced both Mosquitto and MariaDB repositories.
